swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_topics:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      title:
+        $ref: '#/definitions/title'
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          editions:
+            $ref: '#/definitions/Link'
+          latest_edition:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+          latest_version:
+            $ref: '#/definitions/Link'
+
+  EditionsList:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Edition'
+
+  Edition:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      links:
+        type: object
+        properties:
+          self:
+            $ref: '#/definitions/Link'
+          bulletin:
+            $ref: '#/definitions/Link'
+          versions:
+            $ref: '#/definitions/Link'
+          latest_version:
+            $ref: '#/definitions/Link'
+
+
+  VersionsList:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Version'
+
+  Version:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      edition:
+        $ref: '#/definitions/edition'
+      version:
+        $ref: '#/definitions/version'
+      alerts:
+        type: array
+        items:
+          $ref: '#/definitions/Alert'
+      charts:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      correction_notice:
+        type: string
+        example: "A correction has been ma…gise for any inconvenience."
+      downloads:
+        $ref: '#/definitions/Downloads'
+      equations:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      headlines:
+        type: array
+        description: Headlines from this bulletin
+        maxItems: 3
+        items:
+          type: string
+          example: "Headline about this bulletin"
+      images:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      meta_description:
+        type: string
+        example: "Estimates of employment… statistics for the UK."
+      sections:
+        type: array
+        items:
+          $ref: '#/definitions/ContentSection'
+      summary:
+        $ref: '#/definitions/summary'
+      tables:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      update_date:
```

Comment:
I believe the bulletins have a `release_date` rather than an `update_date`.  If a page is updated the only change currently is the alert (which will include a date).